src/tqec/plaquette/plaquette.py

```diff
@@ -58,8 +58,18 @@ def __post_init__(self) -> None:
     def origin(self) -> Position2D:
         return Position2D(0, 0)
 
+    def __eq__(self, rhs: object) -> bool:
+        return (
+            isinstance(rhs, Plaquette)
+            and self.qubits == rhs.qubits
+            and self.circuit == rhs.circuit
+        )
+
+    def __hash__(self) -> int:
+        return hash((self.qubits, str(self.circuit.get_circuit())))
```

Comment:
Note that due to #361 this will eventually need to change to have a reproducible hash.